### ISSUE TITLE
Crash report - runtime error: invalid memory address or nil pointer dereference

### ISSUE THREAD

**Issue by biinari** [github/hub#1282]
Error (runtime.errorString): `runtime error: invalid memory address or nil pointer dereference`

Stack:

```
goroutine 1 [running]:
panic(0x77e480, 0xc420010140)
	/usr/lib/go/src/runtime/panic.go:458 +0x243
github.com/github/hub/github.(*Client).CreatePullRequest(0xc42025bb50, 0xc42011b800, 0xc4200ed2c0, 0xc42025bbb8, 0xc42023caa0, 0x0)
	/tmp/go/src/github.com/github/hub/github/client.go:107 +0x1ed
github.com/github/hub/commands.pullRequest(0x99cd20, 0xc42009e090)
	/tmp/go/src/github.com/github/hub/commands/pull_request.go:214 +0xd92
github.com/github/hub/commands.(*Command).Call(0x99cd20, 0xc42009e090, 0x40b6b9, 0x7f3763)
	/tmp/go/src/github.com/github/hub/commands/commands.go:47 +0x91
github.com/github/hub/commands.(*Runner).Call(0xc420011730, 0x99cd20, 0xc42009e090, 0xc, 0xc420079ca8, 0x450310)
	/tmp/go/src/github.com/github/hub/commands/runner.go:93 +0x3f
github.com/github/hub/commands.(*Runner).Execute(0xc420011730, 0x811620, 0xc420013830, 0xc4200001a0)
	/tmp/go/src/github.com/github/hub/commands/runner.go:85 +0x2cb
main.main()
	/build/hub/src/github-hub-7988c4b/main.go:12 +0x49

```

Runtime:

```
GOOS: linux
GOARCH: amd64
```

Version:

```
git version 2.9.3
hub version 2.2.5
```

Free memory:

```
$ free -m
              total        used        free      shared  buff/cache   available
Mem:          15937        7766        6952         605        1218        7251
Swap:         16383          12       16371
```

**Comment by biinari** [comment]
This error occurred twice in a row for the same attempted pull request, then it went through fine on the third attempt.

Stack trace from first attempt (almost identical) in case it's useful to have both:

```
goroutine 1 [running]:
panic(0x77e480, 0xc420010140)
	/usr/lib/go/src/runtime/panic.go:458 +0x243
github.com/github/hub/github.(*Client).CreatePullRequest(0xc420227b50, 0xc420169400, 0xc4203a6270, 0xc420227bb8, 0xc4203ca120, 0x0)
	/tmp/go/src/github.com/github/hub/github/client.go:107 +0x1ed
github.com/github/hub/commands.pullRequest(0x99cd20, 0xc4200b2090)
	/tmp/go/src/github.com/github/hub/commands/pull_request.go:214 +0xd92
github.com/github/hub/commands.(*Command).Call(0x99cd20, 0xc4200b2090, 0x40b6b9, 0x7f3763)
	/tmp/go/src/github.com/github/hub/commands/commands.go:47 +0x91
github.com/github/hub/commands.(*Runner).Call(0xc420077590, 0x99cd20, 0xc4200b2090, 0xc, 0xc420081d88, 0x450310)
	/tmp/go/src/github.com/github/hub/commands/runner.go:93 +0x3f
github.com/github/hub/commands.(*Runner).Execute(0xc420077590, 0x811620, 0xc42007b5c0, 0xc4200001a0)
	/tmp/go/src/github.com/github/hub/commands/runner.go:85 +0x2cb
main.main()
	/build/hub/src/github-hub-7988c4b/main.go:12 +0x49
```

**Comment by mislav** [comment]
Thanks for reporting. This crash was fixed in[v2.2.6](https://github.com/github/hub/releases/tag/v2.2.6). Please upgrade to latest 2.2.x version.

**Comment by biinari** [comment]
Thanks, I'll update to latest.
